cmd/tnf/fetch/fetch.go

```diff
@@ -15,22 +15,18 @@ import (
 )
 
 var (
-	filterCertifiedOperators = "&filter=organization==certified-operators"
-	containercatalogURL      = "https://catalog.redhat.com/api/containers/v1/images?page_size=%d&page=%d&filter=certified==true"
-	operatorcatalogURL       = "https://catalog.redhat.com/api/containers/v1/operators/bundles?"
-	helmcatalogURL           = "https://charts.openshift.io/index.yaml"
+	containersCatalogSizeURL = "https://catalog.redhat.com/api/containers/v1/images?filter=certified==true&page=0&include=total,page_size,data.docker_image_digest"
```

Comment:
remove data.docker_image_digest, seems not needed

Reply:
You're right. Done!